experiments:

  num_robots: 2
  primitive_file: /home/sameer/masconcomm/src/multiagent/config/robot1.mprim
  primitive_file: /home/sameer/masconcomm/src/multiagent/config/robot2.mprim

  - test: 1
    start: 4.0 5.0 0 1.58
    goal: 6.0 26.0 0 0
    start: 15.5 25.0 0 1.58
    goal: 15.5 5.0 0 0
